modules/local/parse_contamination/main.nf

```diff
@@ -0,0 +1,61 @@
+process PARSE_CONTAMINATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.11' :
+        'biocontainers/python:3.11' }"
+
+    input:
+    tuple val(meta), path(contamination_table)
+
+    output:
+    tuple val(meta), path("*_contamination_mqc.tsv"), emit: mqc_table
+    path "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    #!/usr/bin/env python3
```

Comment:
Can you make this a module binary? We have had issues in the past with some systems interpreting indents differently.